packages/integration-tests/src/tests/api/logto-config.test.ts

```diff
@@ -24,4 +29,22 @@ describe('admin console sign-in experience', () => {
       ...newAdminConsoleConfig,
     });
   });
+
+  it('should get OIDC keys successfully', async () => {
+    const oidcKeys = await getOidcKeys();
+
+    expect(oidcKeys[LogtoOidcConfigKey.PrivateKeys]).toHaveLength(1);
+    expect(oidcKeys[LogtoOidcConfigKey.CookieKeys]).toHaveLength(1);
+
+    expect(oidcKeys[LogtoOidcConfigKey.PrivateKeys][0]).toBeTruthy();
+    expect(oidcKeys[LogtoOidcConfigKey.CookieKeys][0]).toBeTruthy();
+  });
+
+  it('should rotate OIDC keys successfully', async () => {
+    await rotateOidcKeys('rsa');
+    const oidcKeys = await getOidcKeys();
+
+    expect(oidcKeys[LogtoOidcConfigKey.PrivateKeys]).toHaveLength(2);
+    expect(oidcKeys[LogtoOidcConfigKey.CookieKeys]).toHaveLength(2);
```

Comment:
should we check the key type as well instead of only checking its length?

Reply:
Yes, we can. Actually I'm still going to add more API integration test cases to cover more situations, this is like 70% done at the moment.